DOCUMENTATION.md

```diff
@@ -1,810 +1,780 @@
-<summary>
-Click here to expand for more information.
-</summary>
+### Failed
 
-##### `id`
+The only failure status that applies to letters is `technical-failure`. Notify had an unexpected error while sending to our printing provider.
 
-The ID of the notification.
+### Accepted
 
-</details>
+GOV.UK Notify is printing and posting the letter.
 
-## Get the status of all messages
+### Received
 
-#### Method
+The provider has received the letter to deliver.
 
-<details>
-<summary>
-Click here to expand for more information.
-</summary>
+
+## Get the status of one message
+
+You can only get the status of messages that are 7 days old or less.
+
+### Method
 
 ```ruby
-# See section below for a description of the arguments.
-# This will return 250 of the most recent messages if `older_than` is omitted, the following 250 messages can be accessed through the hash `notifications.links["next"]`
-args = {
-  'template_type' => 'sms',
-  'status' => 'failed',
-  'reference' => 'your reference string'
-  'older_than' => 'e194efd1-c34d-49c9-9915-e4267e01e92e' # => Notifications::Client::Notification
-}
-notifications = client.get_notifications(args)
+response = client.get_notification(id)
 ```
 
-</details>
+### Arguments
+
+#### id (required)
+
+The ID of the notification.
+
+### Response
+
+If the request to the client is successful, the client will return a `Notifications::Client::Notification` object. This object is named `response` in the method example.
+
+You can then call different methods on this object to return the requested information:
+
+|Method|Information|Type|
+|:---|:---|:---|
+|`response.id`|Notification UUID|String|
+|`response.reference`| String supplied in `reference` argument|String|
+|`response.email_address`|Recipient email address (email only)|String|
+|`response.phone_number`|Recipient phone number (SMS only)|String|
+|`response.line_1`|Recipient address line 1 of the address (letter only)|String|
+|`response.line_2`|Recipient address line 2 of the address (letter only)|String|
+|`response.line_3`|Recipient address line 3 of the address (letter only)|String|
+|`response.line_4`|Recipient address line 4 of the address (letter only)|String|
+|`response.line_5`|Recipient address line 5 of the address (letter only)|String|
+|`response.line_6`|Recipient address line 6 of the address (letter only)|String|
+|`response.postcode`|Recipient postcode (letter only)|String|
+|`response.type`|Type of notification sent (sms, email or letter)|String|
+|`response.status`|Notification status (sending / delivered / permanent-failure / temporary-failure / technical-failure)|String|
+|`response.template`|Template UUID|String|
+|`response.body`|Notification body|String|
+|`response.subject`|Notification subject (email only)|String|
+|`response.sent_at`|Date and time notification sent to provider|String|
+|`response.created_at`|Date and time notification created|String|
+|`response.completed_at`|Date and time notification delivered or failed|String|
```

Comment:
Response now has one extra method as of today - see https://github.com/alphagov/notifications-ruby-client/pull/45. There's now `response.created_by_name` too (for all notification types). I've documented it in the current docs but I'm sure it could be worded better.